webrtc-stats.html

```diff
@@ -1529,6 +1530,30 @@ <h2>
                   6.3.2.
                 </p>
               </dd>
+              <dt>
+                <dfn>totalPlayoutDelay</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  It is the sum of the time, in seconds, each <a>audio sample</a> or video frame takes from
+                  the time the first RTP packet is received (reception timestamp) and to the time 
+                  the corresponding sample or frame is delivered to the MediaStreamTrack for playout (playout timestamp). 
+                  Given the complexities involved, the time of arrival or the reception timestamp is measured 
+                  as close to the network layer as possible and the playout timestamp is measured as close to
+                  the rendering as possible.
+                  In the case of audio, several samples are received in the same RTP packet, all samples
+                  will share the same reception timestamp and different playout timestamps. 
+                  In the case of video, the frame are received over several RTP packets, in this
+                  case the earliest timestamp containing the frame is counted as the reception timestamp,
+                  and the reception timestamp corresponds to when the complete frame is played out.
+                  This metric is not incremented for frames that are not decoded, 
+                  i.e., {{framesDropped}}, {{partialFramesLost}},
+                  The average playoutDelay delay can be calculated by dividing the {{playoutDelay}} with the
+                  {{totalSamplesReceived}} for audio or {{framesDecoded}} for video.
+                </p>
```

Comment:
It should be clear that totalPlayoutDelay is incremented at the same time as the samples counter, in this time framesDecoded. This implies that the "playout time" is the time that the frames have been decoded.

This part imples that the earlier "[playout] timestamp corresponds to when the complete frame is [ready to be played out]." can be rephrased as "playout timestamp corresponds to when the complete frame is decoded".

If this isn't the intent of the PR I think we need a new frames counter. totalPlayableFrames? Otherwise, all the references to "playout" should change to "decoded"

Reply:
decoded is the correct point of view.

Reply:
added totalSamplesDecoded

Reply:
the new text should align with this